ui/src/utils/feedback.ts

```diff
@@ -0,0 +1,26 @@
+import {PaperId} from "../state";
+import queryString from "querystring";
+
+export function createFeedbackLink(
+  paperId?: PaperId | undefined,
+  extraContext?: object
+) {
+  /*
+   * The URL and field ids below are generated by Google. The identifiers
+   * are opaque to us, which means they could change (say, if we modify
+   * the assocaited form). If this proves to be an issue we'll have to think
+   * through alternative feedback mechanisms.
+   */
+  const baseUrl = "https://forms.gle/BvGeHYzUA1pHzcAK6";
+  const params = {
+    "entry.331961046": JSON.stringify(
+      Object.assign({}, extraContext || {}, { paperId })
+    ),
+  };
```

Comment:
Why not this? It's a bit more readable.
```suggestion
  const params = {
    "entry.331961046": JSON.stringify({
      ...(extraContext || {}),
      paperId,
    }),
  };
```